src/main/steps/applicant1/hub-page/joint/jointTemplateSelector.ts

```diff
@@ -61,6 +63,31 @@ export const getJointHubTemplate = (
     case State.PendingHearingDate: {
       return HubTemplate.PendingHearingOutcome;
     }
+    case State.InformationRequested: {
+      if (isApplicantAbleToRespondToRequestForInformation) {
+        return HubTemplate.InformationRequested;
+      }
+      if (isRequestForInformationForYourPartner) {
+        return HubTemplate.InformationRequestedFromPartner;
+      }
+      break;
+    }
+    case State.AwaitingRequestedInformation:
+      if (isApplicantAbleToRespondToRequestForInformation) {
+        return HubTemplate.AwaitingRequestedInformation;
+      }
+      if (isRequestForInformationForYourPartner) {
+        return HubTemplate.InformationRequestedFromPartner;
+      }
+      break; //Need a hub for awaitingRequestedInfo for different party
```

Comment:
It would be handy if they could use the same page when information is requested from the partner or another party, as we may be able to remove `isRequestForInformationForYourPartner` and have `isApplicantAbleToRespondToRequestForInformation` as our only variable with a ternary operator to select the template

Reply:
Not sure how to handle this scenario.  Kim says we don't need a hub for 'Other' as it won't be used very often, but the 'Partner' hub page is going to cause confusion if 'Other' *is* used (especially if the third party delay responding).  Personally I think we *do* need a hub for Other, even if it's the same as Partner with some variable content.  I'll speak to Kim.

Reply:
We're going to add an additional Hub for 'Other' - I'll speak to Fubara about the wording and try and get it as close to the Partner hub as possible so we can reuse it with some tweaks.